### ISSUE TITLE
Download does not initiate!

### ISSUE THREAD

**Issue by asathu** [FaisalUmair/udemy-downloader-gui#601]
Thanks a lot for this great app "Udeler".
I used it in the past without any issue, but recently an issue has happened for it in both Windows 10 and MacOS.
**When I press on download button, nothing happens! The app stucks at "Building Course Data".** I tried to redownload courses that were downloaded in the past by it, but it is still stucking at that message without progression. I changed the PC and disabled anti-virus. However, the problem is still existing.

So, please, is there any advice to fix the issue?
Regards
Abdullah

**Comment by ScrubDonkey** [comment]
We're unable to download Udemy Courses now. They've implemented DRM so Udeler is useless at this point. It was a nice run.

**Comment by strfr** [comment]
No its downloading easily. Udeler still active.
![image](https://user-images.githubusercontent.com/47529454/122668948-76fc3480-d1c3-11eb-802a-e4524e137626.png)

**Comment by ScrubDonkey** [comment]
> No its downloading easily. Udeler still active.
> ![image](https://user-images.githubusercontent.com/47529454/122668948-76fc3480-d1c3-11eb-802a-e4524e137626.png)

No, it's not, not all courses have been DRM secured. The majority of courses are secured and cannot be downloaded.  I have 2 courses that I can still download.
